### PR TITLE
admin-subject-controller-test

### PR DESCRIPTION
**Pull request by ngovannghiaframgia**
ready

### CODE REVIEW

**Comment by ngovannghiaframgia** [comment]
ready
Screenshot from 2019-03-13 14-06-53

**Comment by ngovannghiaframgia** [comment]
ready

File: nano.save [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (huongnguyenmta) [inline nano.save]: tương tự nè, file rì đây???

**Comment by ngovannghiaframgia** [comment]
ready

**Comment by huongnguyenmta** [comment]
conflicted

**Comment by ngovannghiaframgia** [comment]
ready

**Pull request merged**